core/domain/exp_domain.py

```diff
@@ -653,12 +656,19 @@ def create_default_exploration(
             init_state_name: init_state_dict
         }
 
-        return cls(
+        exploration = cls(
             exploration_id, title, category, objective, language_code, [], '',
             '', feconf.CURRENT_STATE_SCHEMA_VERSION,
             init_state_name, states_dict, {}, [], 0,
             feconf.DEFAULT_AUTO_TTS_ENABLED, False)
 
+        exp_android_proto = cls.to_proto(
```

Comment:
Just to check -- how fast are to_proto() and calculate_size_of_proto(), in milliseconds?

Reply:
This is something new, I will try to check on this how fast these functions are. 
Thank you for this interesting point.

Reply:
`0.00014591217041015625 seconds` = `(time.time() - start_time))`